src/Place/Canonical/PlaceCluster.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\Canonical;
+
+final class PlaceCluster
+{
+    private int $clusterId;
+
+    private array $placeIds;
+
+    public function __construct(int $clusterId, array $placeIds)
+    {
+        $this->clusterId = $clusterId;
+        $this->placeIds = $placeIds;
+    }
+
+    public function getClusterId(): int
```

Comment:
At first glance this extra class looked like a good abstraction, but now after inspecting the code the method `getClusterId` is nowhere used and `getPlacesIds` just returns an array of strings. So I think the method `DuplicatePlaceRepository::getCluster` could just return a string array.